### DISCUSSION TITLE
Are there any documented benchmarks for SpiceDB cloud services?

### DISCUSSION THREAD

**Discussion by ilterpehlivan** [authzed/spicedb#1507] [Q&A] [answered]
Hi,

Do you have any benchmarking done in cloud ? If so; any written document for this results ?

Thanks

**Comment by corkrean** [comment, marked as answer]
Hi @ilterpehlivan, 

You can see the results of a performance test of SpiceDB [here](https://authzed.com/blog/google-scale-authorization).

You can expect similar results in SpiceDB Dedicated.

**Reply by ilterpehlivan** [reply]
thank you for quick response

BR
